Include msisdns in generated report

smn opened the pull request.

smn committed:
include msisdns in generated report

smn committed:
fix tests

smn committed:
fix tests for reals now

smn commented:
@DevChima ready for review!

DevChima reviewed (commented):
Python newbie here who's just curious:
Please what's the purpose of the function 'status=200' in this context. 
I kind of understand that it's a HTTP status code but shouldn't it be a condition for an action.
Ex:
if status=200 THEN do something.

smn commented on ci/tests.py:
```diff
@@ -110,8 +110,14 @@ def test_generate_report(self):
+                    }
                 }
             },
             status=200,
```
@DevChima did you mean this `status=200`?

This is our testing framework, essentially we're mocking responses to HTTP requests our applications make as part of a test. So instead of actually talking to the identity store api we capture that request and have the test framework reply with the payload and an HTTP Status of 200. The application then thinks this is the response it gets back from the identity store api and continues. That way we can control what is received as input to the application and predict (and test) the outcomes.

DevChima replied:
Thanks @smn. That clears it up.
At first I didn't realize it was a testing environment (I'm just noticing the Asserts).
What else am I supposed to do here? Approve merging changes?

smn replied:
@DevChima mostly follow along and ask question for stuff that strike you as odd or don't understand :)

smn committed:
collect msisdn from the receiver record, not the operator

smn commented:
@DevChima I've added some more changes now. It turns out we were using the record of the operator (in this case that is the HCW) to retrieve the MSISDN from. This gave us multiple registrations for the same number as the operator is registering multiple women. I've no changed this to retrieve the mother's record from the identity store (identified by the `receiver_id`) and am retrieving the MSISDN from that record. You'll notice that I added another mocked HTTP response for the request retrieving the mothers `receiver_id` and am using the MSISDN I specified in that response to test whether or not the same MSISDN is written to the Excel sheet.

DevChima commented:
Understood Simon. So the second part is a check for duplicate MSISDNs in the excel sheet.
"....and am using the MSISDN I specified in that response to test whether or not the same MSISDN is written to the Excel sheet"

smn commented:
yup! happy for me to merge this pull request into the existing code base?

smn commented:
Merging as per Jira ticket discussion @DevChima
